pyglmnet/datasets.py

```diff
@@ -57,23 +88,35 @@ def fetch_tikhonov_data(dpath):
     dpath : str
         The data path
     """
+    try:
+        import pandas as pd
+    except ImportError:
+        raise ImportError('The pandas module is required for reading the '
+                          'community crime dataset')
 
-    if os.path.exists(dpath):
-        shutil.rmtree(dpath)
-    os.mkdir(dpath)
-
-    base_url = "https://raw.githubusercontent.com/glm-tools/datasets/master"
+    dpath = get_data_home(data_home=dpath)
     fnames = ['fixations.csv', 'probes.csv', 'spiketimes.csv']
 
-    for fname in fnames:
-        url = base_url + "/tikhonov/" + fname
-        fname = os.path.join(dpath, fname)
-        urlretrieve(url, fname, _reporthook)
+    if not (op.isdir(dpath) and all(op.exists(op.join(dpath, fname))
+                                    for fname in fnames)):
+        base_url = (
+            "https://raw.githubusercontent.com/glm-tools/datasets/master"
+        )
+        fnames = ['fixations.csv', 'probes.csv', 'spiketimes.csv']
+
+        for fname in fnames:
+            url = base_url + "/tikhonov/" + fname
+            fname = os.path.join(dpath, fname)
+            urlretrieve(url, fname, _reporthook)
 
-    return dpath
+    fixations_df = pd.read_csv(op.join(dpath, 'fixations.csv'))
+    probes_df = pd.read_csv(op.join(dpath, 'probes.csv'))
+    spikes_df = pd.read_csv(op.join(dpath, 'spiketimes.csv'))
 
+    return fixations_df, probes_df, spikes_df
```

Comment:
let's keep the function signature similar for all fetchers. They should all return the data path.